Example/Backpack/View Controllers/IconsViewController.swift

```diff
@@ -77,8 +83,20 @@ extension IconsViewController {
                 fatalError("No cell registered for reuse with identifier \(IconsViewController.cellIdentifier)")
         }
 
-        let icon = IconsViewController.icons[indexPath.row]
-        cell.icon = icon
+        let iconSet = IconsViewController.icons[indexPath.section]
+
+        // Whether we set `size` or `icon` first, we will potentially be trying to use an icon that doesn't exist.
+        // By setting the icon to `accessibility`, then setting the size, and then setting the iconName, we should
+        // never be trying to use an icon at an unavailable size.
```

Comment:
This is a bit of a hacky workaround. Fine for an example app, but maybe we should just reduce the severity of the error if the icon name and size combination doesn't exist, otherwise consumers might have to start doing similar things in their code

Reply:
We could provide a method that can veirfy if a given icon is available at a given size?

Reply:
That won't actually help though, because by definition you still have to change one property before the other

Reply:
Hmm yeah I guess this speaks even more to the idea that we should encode the size in the enums